.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,17 @@
 <!--
-    As we are working towards a stable version of uuid, we require that you 
-    open an issue, before submitting a pull request. If the pull request is 
-    imcomplete, prepend the Title with WIP: 
+    If this PR is a breaking change, ensure that you are opening it against 
+    the `breaking` branch.  If the pull request is imcomplete, prepend the Title with WIP: 
 -->
 
 **I'm submitting a ...**
-  - [ ] bug fix
-  - [ ] feature enhancement
-  - [ ] deprecation or removal
-  - [ ] refactor
+<!-- Feature|Bug|Refactor|Enhancement..., -->
+
 
 # Description
-<!-- Provide a summary of your changes in the Title above -->
 
 # Motivation
-<!-- Why is this change required -->
 
 # Tests
 <!-- How are these changes tested? -->
 
 # Related Issue(s)
-<!-- 
```

Comment:
This can be added back with the following adjustment:

```
<!--
  Unless you are making a minor change, that DOES NOT change the public API, we require an issue for every PR.
  Link the Issue here
-->
```